.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  generator-jhipster-nav-element:
+    name: npm test
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
remove this please

Reply:
Are you sure? it is only a useful condition if you want to skip the automatic build, that is trigger only if you include in your commit message the **ci skip* suffix. Otherwise, it is ignored.

Reply:
I know what it does, currently I don't have a use case for ignoring the ci.
This is very JHipster specific, their ci builds are very long running so it makes sense to have it there. 
Currently this is not needed in this module.

Reply:
Ok, I will remove it.